tests/Utils/BuildClientSchemaTest.php

```diff
@@ -0,0 +1,1082 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GraphQL\Tests\Utils;
+
+use GraphQL\GraphQL;
+use GraphQL\Type\Definition\EnumType;
+use GraphQL\Type\Definition\ObjectType;
+use GraphQL\Type\Definition\Type;
+use GraphQL\Type\Introspection;
+use GraphQL\Type\Schema;
+use GraphQL\Utils\BuildClientSchema;
+use GraphQL\Utils\BuildSchema;
+use GraphQL\Utils\SchemaPrinter;
+use PHPUnit\Framework\TestCase;
+use function array_filter;
+
+/**
+ * @see BuildClientSchema
```

Comment:
I'd change it to

```suggestion
 * @covers GraphQL\Utils\BuildSchema
```

It will prevent coverage leaking and have same documentation effect.

Reply:
I am not convinced that using `@covers` is necessary. There seems to be very little gain for a bunch of work.

Reply:
What do you mean by bunch of work? It's basically just change of one line, isn't it? Can you explain?

I discovered this recently and I like how the coverage does not leak. When I generate line coverage it show much better what I covered in tests and what just "accidentally" runs. But whether something runs or not I know from PHPStan already so personally I prefer not to have unit B covered by testing unit A.

Reply:
I don't see the problem with "leaking coverage".

Reply:
The coverage is partially fabricated so the metric is less accurate and makes detection of untested paths harder.

Reply:
The code paths are being tested though, albeit indirectly.

Reply:
IMO there's no indirect testing, either you test it or not. Running the code is not testing.  
Whether coverage leaking is issue depends on whether classes it leaks to are part of the unit or not. But those classes used in BuildClientSchema are not part of the unit so coverage should not be leaking to them.

This is just the nitpicking though. I saw the `@see` annotation so having `@covers` there would make more sense to me.  AFAIK we do not use this annotation here yet.